keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1088,10 +1240,18 @@ type Device struct {
     // +optional
     Attributes []DeviceAttribute
 
-    // TODO for 1.31: define how to support partitionable devices
+    // SharedCapacityConsumed defines the set of shared capacity consumed by
+    // this device.
+    //
+    // Must not have more than 32 entries.
+    //
+    // +listType=atomic
+    // +optional
+    SharedCapacityConsumed []SharedCapacity
 }
 
 const ResourceSliceMaxAttributesPerDevice = 32
+const ResourceSliceMaxSharedCapacityConsumedPerDevice = 32
```

Comment:
Except that it's 7 per MIG (https://github.com/pohly/enhancements/pull/18#discussion_r1636625517), so it would be only 18. Perhaps we have to keep ResourceSliceMaxSharedCapacity at 256?

Reply:
We don't need to finalize the decision, but the KEP should at least explain the choices and consequences, so I'm also fine with the current numbers.

Reply:
Each device will only have 7 entries, indicating how much capacity it consumes from each.

Reply:
Or right, the memroy blocks aren't a single entry anymore, so it becomes 6+8 = 14 entries per device

Reply:
I am thinking of ResourceSliceMaxSharedCapacity = 128. That allows defining the blocks of 128 / 14 = 9 MIGs per node, if I am not mistaken. Seems a bit low?

Reply:
Updated.

Reply:
I'm not sure what you mean by `9 MIGs per node`. I can model up to 9 full GPUs, each of which has 15 possible devices that can be built from that (1 full GPU and 14 MIG devices).

Reply:
I was thinking of the size of the `slice.spec.sharedCapacity` slice. All MIGs need to put "their" capacities there, so depending on how many are needed per MIG, that could have been the limiting factor - but it wasn't, because you already hit the `slice.spec.devices` limit.